test/common/http/client_test.cc

```diff
@@ -5,6 +5,7 @@
 #include "common/stats/isolated_store_impl.h"
```

Comment:
Are unit tests for the provisional dispatcher coming in a subsequent PR?

Reply:
Yes